src/pages/events/division.tsx

```diff
@@ -20,18 +20,23 @@ export const EventDivisionPickerPage: React.FC = () => {
 
   return (
     <section className="mt-4 flex flex-col gap-4">
-      <ol className="contents">
+      <ol className="overflow-auto max-h-screen">
```

Comment:
`display: contents` here is a bit of a performance optimization to reduce the number of distinct flexboxes/grids in the UI at once (which makes layout much simpler on bad devices). If we can get away with keeping it, that would be helpful (although I am not sure it is doing much here)